app/jobs/job_archiver_job.rb

```diff
@@ -0,0 +1,27 @@
+class JobArchiverJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    archive_jobs
+  end
+
+  def archive_jobs
+    @jobs = Job.all
+
+
+    @jobs.each do |j|
+      if DateTime.now > j.exp_date
```

Comment:
This if block is redundant try and rewrite it to be a one liner and read more like english.

[Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#no-nested-modifiers)